scripts/closure/closure_compiler.sh

```diff
@@ -0,0 +1,57 @@
+OPTS=(
+  "--compilation_level=ADVANCED_OPTIMIZATIONS"
+  "--js_output_file=build/closure/closure-bundle.js"
+  "--rewrite_polyfills=false"
+
+  # Flags to simplify debugging.
+  "--formatting=PRETTY_PRINT"
+
+  "--externs=dist/zone_externs.js"
+)
+
+OPTS_FAILED=(
+  "--compilation_level=ADVANCED_OPTIMIZATIONS"
+  "--js_output_file=build/closure/closure-bundle.js"
+  "--rewrite_polyfills=false"
+
+  # Flags to simplify debugging.
+  "--formatting=PRETTY_PRINT"
+
+  "--externs=test/closure/zone_externs_failed.js"
```

Comment:
nit: I'd have one options block with the options that are in common (including the zone.closure.js input file)
then when you call compiler.jar below, give the options that differ between the two cases